docs/src/SUMMARY.md

```diff
@@ -57,12 +57,24 @@
     * [clean-cache](appendix/snforge/clean-cache.md)
 * [Cheatcodes Reference](appendix/cheatcodes.md)
     * [CheatSpan](appendix/cheatcodes/cheat_span.md)
-    * [caller_address](appendix/cheatcodes/caller_address.md)    
+    * [caller_address](appendix/cheatcodes/caller_address.md)
     * [block_number](appendix/cheatcodes/block_number.md)
     * [block_timestamp](appendix/cheatcodes/block_timestamp.md)
-    * [mock_call](appendix/cheatcodes/mock_call.md)
-    * [tx_info](appendix/cheatcodes/tx_info.md)
     * [sequencer_address](appendix/cheatcodes/sequencer_address.md)
+    * [version](appendix/cheatcodes/transaction_version.md)
+    * [account_contract_address](appendix/cheatcodes/account_contract_address.md)
+    * [max_fee](appendix/cheatcodes/max_fee.md)
+    * [signature](appendix/cheatcodes/signature.md)
+    * [transaction_hash](appendix/cheatcodes/transaction_hash.md)
+    * [chain_id](appendix/cheatcodes/chain_id.md)
+    * [nonce](appendix/cheatcodes/nonce.md)
+    * [resource_bounds](appendix/cheatcodes/resource_bounds.md)
+    * [tip](appendix/cheatcodes/tip.md)
+    * [paymaster_data](appendix/cheatcodes/paymaster_data.md)
+    * [nonce_data_availability_mode](appendix/cheatcodes/nonce_data_availability_mode.md)
+    * [fee_data_availability_mode](appendix/cheatcodes/fee_data_availability_mode.md)
+    * [account_deployment_data](appendix/cheatcodes/account_deployment_data.md)
+    * [mock_call](appendix/cheatcodes/mock_call.md)
```

Comment:
those could be rolled into a section of their own

Reply:
We tried, mdBook is not ok with that, there can only be links